server/routes/index.js

```diff
@@ -5,7 +5,9 @@ const express = require("express");
 const { runHealthChecks } = require("../health");
 const { log } = require("../logger");
 const { compileSchemaString } = require("../gql/helper");
-const { schema } = require("../models");
+const { postgresConfig } = require("../config");
+const { GraphQLSchema } = require("@aerogear/data-sync-gql-core").models(postgresConfig);
```

Comment:
Why we do not use getModels from api? This seems to be initializing models for every file. 
We could create single module here that is doing that once or setup core based on postgresConfig.